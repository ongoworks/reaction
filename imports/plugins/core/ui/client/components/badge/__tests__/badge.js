jest.mock("/imports/plugins/core/ui/client/components", () => {
  return {
    Translation(props) {
      return <span>{props.defaultValue}</span>; // eslint-disable-line
    }
  };
});

import React from "react";
import Badge from "../badge";
import { shallow } from "enzyme";
import shallowToJSON from "enzyme-to-json";

/**
 * Order Summary is a display only component
 * It receives props and displays them accordingly
 */

afterEach(() => {
  jest.clearAllMocks();
});

/**
 * Snapshots make sure your UI does not change unexpectedly
 */

test("Badge snapshot test", () => {
  const component = shallow(
    <Badge
      label="Badge Label"
    />
  );
  const tree = shallowToJSON(component);
  expect(tree).toMatchSnapshot();
});
