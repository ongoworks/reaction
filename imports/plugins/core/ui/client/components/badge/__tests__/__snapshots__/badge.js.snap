// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Badge snapshot test 1`] = `
<span
  className="rui badge badge-small badge-default"
  style={
    Object {
      "display": "inline-flex",
    }
  }
>
  <Translation
    defaultValue="Badge Label"
  />
</span>
`;
